src/antlr/Lexer.g

```diff
@@ -56,7 +56,8 @@ lexer grammar Lexer;
 }
 
 // Case-insensitive keywords
-// When adding a new reserved keyword, add entry to o.a.c.cql3.ReservedKeywords as well
+// When adding a new reserved keyword, add entry to o.a.c.cql3.ReservedKeywords and
+// pylib/cqlshlib/cqlhandling.py::CqlParsingRuleSet.cql_keywords_reserved as well
```

Comment:
Nit: I think it's `pylib/cqlshlib/cqlhandling.py::cql_keywords_reserved`

Reply:
Fixed in https://github.com/apache/cassandra/pull/1105/commits/489fe9db380d055e2b350b2c457a1fd6e0e4859b#diff-30dc21e0dca41aab4d29072667b449dc39157d645ebba1297d0ca12ef62b713eL58-R61.